src/event.cxx

```diff
@@ -81,10 +81,7 @@ typedef struct {
   Sint16 reserve2[88];
 } DescInfo;
 
-// Toutes les premières lettres doivent
-// être différentes !
-
-static char cheat_code[9][20] = {
+static char cheat_code[MAXCHEAT][CHEATLENGTH] = {
```

Comment:
It's a very old and ugly original code. You should prefer "modern" stuff like std::array/std::vector with std::string when replacing old stuff.